ci/ciimage/bionic/install.sh

```diff
@@ -58,6 +58,22 @@ update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 2 \
 
 python3 -m pip install -U "${base_python_pkgs[@]}" "${python_pkgs[@]}"
 
+pushd /opt
+# Download and install PyPy3.8, and link it to /usr/bin/pypy3
+# At some point it would be more robust to download and parse
+# https://downloads.python.org/pypy/versions.json
+wget https://downloads.python.org/pypy/pypy3.8-v7.3.9-linux64.tar.bz2
```

Comment:
Is there e.g. some sort of "latest" style url download available? That might save parsing a json file I suppose.

Reply:
Adding a symlink to a https://downloads.python.org/pypy/pypy3.8-latest-linux64.tar.bz2 is a nice idea. Maybe if we move to signed tarballs, then you could verify the signature.